roles/db/tasks/gateway.yml

```diff
@@ -3,15 +3,12 @@
 - name: Create Gateway database
   mysql_db: name={{ database_gateway_name }} state=present login_user=root login_password={{ mariadb_root_password | vault(vault_keydir) }}
 
-- name: Create Gateway test database
-  mysql_db: name=gateway_test state=present login_user=root login_password={{ mariadb_root_password | vault(vault_keydir) }}
-
 - name: Create gateway user
   mysql_user:
     name: "{{ database_gateway_user }}"
     host: "%"
     password: "{{ database_gateway_password | vault(vault_keydir) }}"
-    priv: "{{ database_gateway_name }}.*:SELECT/gateway_test.*:SELECT"
+    priv: "{{ database_gateway_name }}.*:SELECT"
```

Comment:
Regarding the revert of my previous `Provide test databases` commit (68653b9). Personally, I would have added a revert commit, or remove said commit entirely from this PR. That might result in a cleaner git history in the long run.